src/Microsoft.OData.Core/IServiceCollectionProvider.cs

```diff
@@ -9,14 +9,14 @@
 namespace Microsoft.OData
 {
     /// <summary>
-    /// An interface that provides a dependency injection container.
+    /// An interface that provides a dependency injection service collection.

```

Comment:
`An interface that provides a dependency injection <see cref="IServiceProvider" />.`